pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -175,6 +176,33 @@ Avo will override all events in the RudderStack Tracking Plan that are being imp
 
 Avo will not override any events that are only defined in RudderStack, or events that have not been excluded from the RudderStack integration in Avo with filters.
 
+<img
+  src={require('../../../images/publishing_mparticle.svg')}
+  alt="mParticle"
+  style={{ margin: '32px 0 -32px 0' }}
+/>
+
+## <a name="mparticle"></a> mParticle Data Master
+
+The [mParticle integration](https://docs.mparticle.com/guides/data-master/introduction/) enables you to publish your Avo Tracking Plan into mParticle. You can configure the integration to publish whenever a branch is merged or with a push of a button.
+
+### Configure the mParticle integration
+
+To enable the direct integration with mParticle Data Master, you need to configure it with the following parameters from mParticle.
+
+- **mParticle Client ID**: Your client ID issued by mParticle
+- **mParticle Client Secret**: Your client secret issued by mParticle
+- **mParticle Workspace ID**: The ID of the workspace containing your data plans
+- **mParticle Plan ID**: The ID of the Data Plan to update
```

Comment:
Can we add more details here on how you find these things in the mParticle UI? I had a hard time finding the workspace id at first, and the client id/secret also requires a bit of work.

Reply:
even a hyperlink to the location would be nice if that is possible